pkgs/top-level/python-packages.nix

```diff
@@ -4809,6 +4809,8 @@ in {
 
   oyaml = callPackage ../development/python-modules/oyaml { };
 
+  packageurl = callPackage ../development/python-modules/packageurl-python { };
```

Comment:
```suggestion
  packageurl-python = callPackage ../development/python-modules/packageurl-python { };
```

Reply:
Sorry, but no. In a previous version of this PR I was asked to rename it from "packageurl-python" to "packageurl" ;-)

Reply:
Didn't know that. I would say @rmcgibbo was wrong there, but maybe @SuperSandro2000, @FRidh, or @jonringer have an opinion on that. I thought the attribute name should always match `pname`.

Reply:
> Didn't know that. I would say @rmcgibbo was wrong there, but maybe @SuperSandro2000, @FRidh, or @jonringer have an opinion on that. I thought the attribute name should always match `pname`.

Personally I don't care what it is called but you should all make up your mind :-)

Reply:
The normalized PyPI name for both, so `packageurl-python`. This name is predictable and can be used by external tooling such as mach-nix.

Reply:
clarifying this in https://github.com/NixOS/nixpkgs/pull/116344

Reply:
Sorry for the back and forth @armijnhemel!